src/main/java/edu/ucsb/ucsbcslas/controllers/TutorAssignmentController.java

```diff
@@ -271,5 +271,19 @@
       String body = mapper.writeValueAsString(newAssignment);
       return ResponseEntity.ok().body(body);
     }
-  
+    
+    @DeleteMapping (value = "/api/member/tutorAssignments/{id}", produces = "application/json")
+    public ResponseEntity<String> deleteTutorAssignment(@RequestHeader("Authorization") String authorization,
+        @PathVariable("id") Long id) throws JsonProcessingException {
+
+      if (!authControllerAdvice.getIsAdmin(authorization))
+        return getUnauthorizedResponse("admin");
+      Optional<TutorAssignment> tutorAssignment = tutorAssignmentRepository.findById(id);
+      if (!tutorAssignment.isPresent()) {
+        return ResponseEntity.notFound().build();
+      }
+    
+      tutorAssignmentRepository.deleteById(id);
+      return ResponseEntity.noContent().build();
+    }
```

Comment:
This looks ok, and should work.  I would suggest starting a new branch with ONLY this change, and nothing else.    You can test it through swagger-ui/index.html.     I can show you how in office hours.

This will be easier to merge if we focus on JUST the one change, and tests for this one change, with no front end code: a pure backend change.  Then we can return to this branch, and work on hooking up the front end to the backend in a separate story.   Hopefully, you get points for both, but it's better to get points for one of them, then lose all the points for both.